deploy/catalog_resources/redhat/7.8/businessautomation-operator.7.8.0.clusterserviceversion.yaml

```diff
@@ -211,8 +209,6 @@ spec:
                   value: registry.redhat.io/rhpam-7/rhpam-businesscentral-rhel8:7.7.0
                 - name: PAM_BC_MONITORING_IMAGE_7.7.0
                   value: registry.redhat.io/rhpam-7/rhpam-businesscentral-monitoring-rhel8:7.7.0
-                - name: PAM_PROCESS_MIGRATION_IMAGE_7.7.0
```

Comment:
shouldn't be here... did you rebase these files as well? run `make csv` after and add to your commit?

Reply:
nm, i see now that this wasn't removed for 7.8.0 CSV's with prior cherry-pick... so this is expected. all good...